tests/integration/MyObservations.test.js

```diff
@@ -11,57 +11,80 @@ import factory, { makeResponse } from "../factory";
 import { renderAppWithComponent } from "../helpers/render";
 import { signIn, signOut } from "../helpers/user";
 
-jest.useFakeTimers( );
+jest.useFakeTimers();
 
-describe( "MyObservations", ( ) => {
-  beforeEach( signOut );
+jest.mock( "i18next", () => {
+  const originalModule = jest.requireActual( "i18next" );
 
-  afterEach( ( ) => {
-    jest.clearAllMocks( );
-  } );
+  return {
+    // Ideally we shouldn't have to test i18next itself
+    // just need to test that the correct keys/opts are being fed to i18next
+    t: ( key, options ) => {
+      if ( !options ) {
+        return key;
+      }
+      return originalModule.t( key, options );
+    }
+  };
```

Comment:
I feel this is a kind of hacky solution to ensure the localization tests still pass.